const handleError = (error, req, res, next) => {
    console.log(error)
}

module.exports = handleError